### PR TITLE
feat: added FrameWallet

### PR DESCRIPTION
**Pull request by FinsaasGH**

### CODE REVIEW

File: configs/dapps.js [hunk 1]
```diff
@@ -431,6 +431,16 @@ export const categoriesData = [
...
+      {
+        id: 7,
+        name: 'FrameWallet',
+        description: 'A privacy focused Ethereum wallet that runs natively on macOS, Windows and Linux',
+        favorite: false,
+        link: 'https://frame.sh/',
+        icon: {
+          url: './assets/images/dapps/dapp_FrameWallet.png',
+        },
+      },
```
Comment (kauri-hero) [inline configs/dapps.js:443]: going to add to the `Discover` category also to showcase Frame wallet when users first head to dApp Library

**Review by kauri-hero — APPROVED** [review]

**Review by kauri-hero — CHANGES_REQUESTED** [review]
sorry my last review I selected 'Approve'

please see small adjustment for `dapps.js`

File: configs/dapps.js [hunk 2]
```diff
@@ -48,6 +48,16 @@ export const categoriesData = [
       },
       {
         id: 4,
+        name: 'FrameWallet',
```
Comment (kauri-hero) [inline configs/dapps.js:51]: Sorry one last change - can you change name to just 'Frame'

File: configs/dapps.js [hunk 3]
```diff
@@ -431,6 +441,16 @@ export const categoriesData = [
           url: './assets/images/dapps/dapp_CoinGecko.png',
         },
       },
+      {
+        id: 7,
+        name: 'FrameWallet',
```
Comment (kauri-hero) [inline configs/dapps.js:446]: Same here please

**Review by kauri-hero — CHANGES_REQUESTED** [review]
just simple name change

**Comment by FinsaasGH** [comment]
changes completed

**Review by kauri-hero — APPROVED** [review]

**Pull request merged**